app/src/main/java/org/p2p/wallet/home/ui/select/NewSelectTokenViewHolder.kt

```diff
@@ -0,0 +1,72 @@
+package org.p2p.wallet.home.ui.select
+
+import android.graphics.drawable.PictureDrawable
+import android.net.Uri
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.core.view.isVisible
+import com.bumptech.glide.Glide
+import com.bumptech.glide.RequestBuilder
+import com.bumptech.glide.request.RequestOptions
+import org.p2p.uikit.glide.SvgSoftwareLayerSetter
+import org.p2p.wallet.common.ui.recycler.adapter.BaseSelectionViewHolder
+import org.p2p.wallet.databinding.ItemNewTokenSimpleBinding
+import org.p2p.wallet.home.model.Token
+import org.p2p.wallet.utils.viewbinding.inflateViewBinding
+import java.util.Locale
+
+class NewSelectTokenViewHolder(
+    parent: ViewGroup,
+    binding: ItemNewTokenSimpleBinding = parent.inflateViewBinding(attachToRoot = false),
+    private val onItemClicked: (Token) -> Unit
+) : BaseSelectionViewHolder<Token>(binding.root, onItemClicked) {
+
+    companion object {
+        private const val IMAGE_SIZE = 56
+    }
+
+    private val requestBuilder: RequestBuilder<PictureDrawable> = Glide.with(binding.root.context)
+        .`as`(PictureDrawable::class.java)
+        .listener(SvgSoftwareLayerSetter())
+
+    private val tokenImageView = binding.tokenImageView
```

Comment:
it would be painful if we change layout ids in binding, and then we need to change fields name too
i suggest just to use binding, looks like overhead